### ISSUE TITLE
Pill color (low, medium) don't pass a11y contrast ratio

### ISSUE THREAD

**Issue by oscarkwan** [acl-services/paprika#427]
# Bug Report
## Expected behavior
- low and medium pill colors should have a 4.5 contrast ratio

## Current behavior
**Low pill:**
Element has insufficient color contrast of 3.5 (foreground color: #ffffff, background color: #299a7a, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

**Medium pill:**
Element has insufficient color contrast of 2.17 (foreground color: #ffffff, background color: #c9af28, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

## Screenshots / Gifs / Codepens
![Screen Shot 2020-02-18 at 11 16 48 AM](https://user-images.githubusercontent.com/19940076/74769950-2c218400-5240-11ea-862a-01ae64354644.png)

**Comment by oscarkwan** [comment]
cc: @sunminsarahlee @johnpeele , probably need help with getting the right colours.

Low/Medium/High are actually special colour codes that aren't in our tokens. Maybe we need to get rid of these?

**Comment by johnpeele** [comment]
@oscarkwan @sunminsarahlee I am not as familiar with the color requirements for these pills and the difference between low/medium/high. I didn't want to ignore the issue, but I'm not sure how I can be helpful.

**Comment by oscarkwan** [comment]
I'll talk to sarah when i get a chance

**Comment by mikrotron** [comment]
@sunminsarahlee has ruled that:
> risk colours would be exceptional case. those colours are used for risk scoring in the industry. we can leave it as is for now.
